src/pages/landing/stocks/_whyTradeWIthUs.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Container, Flex, SectionContainer } from 'components/containers'
+import { Header, Text } from 'components/elements'
+import device from 'themes/device.js'
+
+const Wrapper = styled(Flex)`
+    flex-wrap: wrap;
+    flex-direction: row;
+    @media ${device.tabletL} {
+        flex-direction: column;
+        justify-content: center;
+    }
+`
+
+const ImageWrapper = styled.img`
```

Comment:
I think `StyledImage` might be a little more appropriate here.

Reply:
Changed to StyledImage.